config/config.exs

```diff
@@ -0,0 +1,4 @@
+import Config
+
+config :scidata, 
+  max_concurrency: "MAX_CONCURRENCY" |> System.get_env("#{System.schedulers_online}") |> String.to_integer
```

Comment:
Note this configuration only affects scidata itelf. It is not used by actual projects. In any case, avoiding global configuration is recommended, so we should prefer options, as I left a note below. :)